modules/0_git/test/main.cpp

```diff
@@ -4,10 +4,12 @@
 
 #include "header.hpp"
 
+
 TEST(git, say_hello) {
     myspace::A a;
     EXPECT_EQ(myspace::func(a), "Hello, Nizhny!");
-    EXPECT_EQ(func(a), "Hello, World!");
+    EXPECT_EQ(func(a), "Hello, Nizhny!");
```

Comment:
Please create new branch and don't add old task

Reply:
I didnt quite understand. I created new branch and made new commit. Do I need to clon code for each job?

Reply:
Master branch doesn't have yours source code. If you will create new task based on master branch then old task will not show in github differences. You can local check differences with help `git diff`.

Reply:
I need change branch on master then create new branch and make commit. Do I understand correctly?

Reply:
You need to synchonize master branch yours fork and master branch `openvino_practice` and then create new branch

Reply:
Just do
```
git checkout master
git checkout -b new_task
```
before new module - than commit the changes

Reply:
I did. Do I need create new PR?

Reply:
Yes, for a new branch